src/components/deal-proposal-form/DurationInput.tsx

```diff
@@ -0,0 +1,108 @@
+import { type Control, Controller, type FieldError, type Path } from "react-hook-form";
+import type { FormValues } from "./types";
+
+interface DurationFieldProps {
+  name: Path<FormValues>;
+  label?: string;
+  required?: boolean;
+  defaultValue?: DurationValue;
+  disabled?: boolean;
+  className?: string;
+  maxMonths?: number;
+  maxDays?: number;
+  control: Control<FormValues>;
+  error?: FieldError;
+}
+
+export interface DurationValue {
+  months: number;
+  days: number;
+}
+
+const DurationInput: React.FC<DurationFieldProps> = ({
+  name,
+  label,
+  control,
+  error,
+  required = false,
+  defaultValue = { months: 0, days: 0 },
+  disabled = false,
```

Comment:
This is always false and we're not changing it ever, we can *not* expose the parameter